packages/sit-onyx/src/styles/z-indices.scss

```
@@ -0,0 +1,9 @@
+// all z-indices used in the onyx design system
+:root {
+  --onyx-z-index-overlay: 60;
+  --onyx-z-index-top-nav: 50;
+  --onyx-z-index-page-loader: 40;
+  --onyx-z-index-global-info: 30;
+  --onyx-z-index-flyout: 20;
+  --onyx-z-index-tooltip: 10;
```

```suggestion
  --onyx-z-index-overlay: 60;
  --onyx-z-index-navigation: 50;
  --onyx-z-index-page-overlay: 40;
  --onyx-z-index-page-raised: 30;
  --onyx-z-index-flyout: 20;
  --onyx-z-index-popover: 10;
```

I would like to decouple the z-index from specific components. Do we need a z-index for flyout and tooltip? If we implement them with the Popover/Dialog API they will be placed in the [CSS top layer](https://developer.mozilla.org/en-US/docs/Glossary/Top_layer)

---

Please see https://github.com/SchwarzIT/onyx/issues/108#issuecomment-1952587571

---

renaming will be discussed with Jannick, he was not available  :(